index.js

```diff
@@ -0,0 +1,3 @@
+const reporter = require('./reporter/index');
```

Comment:
You can remove the `/index`
```suggestion
const reporter = require('./reporter');
```